ChangeLog

```diff
@@ -71,6 +71,8 @@ Features
      independently contributed again by Paul Sokolovsky.
    * Add support for key wrapping modes based on AES as defined by
      NIST SP 800-38F algorithms KW and KWP and by RFC 3394 and RFC 5649.
+   * Add support for 128-bit keys in CTR_DRBG. Note that using keys shorter
+     than 256 bits limits the security of generated material to 128 bits.
```

Comment:
Please add: “Fixes #1146.”, with a commit message containing “fix #1146” so that Github will close the issue when this PR is merge.